static/languages/he/qanda.json

```diff
@@ -17,5 +26,7 @@
     "admin.form.general_settings": "הגדרות כלליות",
     "admin.form.label.only_allow_all": "אפשר שאלות בלבד בכל הקטגוריות (בטל נושאים רגילים)",
     "admin.form.label.only_allow_following": "אפשר שאלות בלבד בקטגוריות המסומנות (בטל נושאים רגילים)",
-    "admin.form.tips": "הגדרה זו עוקפת את ההגדרה למטה."
+    "admin.form.tips": "הגדרה זו עוקפת את ההגדרה למטה.",
+    "admin.form.label.toggle_loc": "נעל אוטומטית נושאים שנפתרו",
```

Comment:
should be `admin.form.label.toggle_lock`

Reply:
Oops... thanks for checking, fixed